### ISSUE TITLE
Error Spamming in Console

### ISSUE THREAD

**Issue by Lucien950** [jagnani73/react-easy-marquee#13]
**Describe the bug**
Something in the CSS is causing a error

**To Reproduce**
Use the module????

**Expected behavior**
No error in console

**Screenshots**
![image](https://user-images.githubusercontent.com/20777515/142769878-72b4c30c-7285-461e-bd78-72093cb2c809.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 96.0.4664.45

**Comment by Lucien950** [comment]
Note: I am using NextJS

**Comment by Lucien950** [comment]
Also, here is the console output in text
```
react_devtools_backend.js:2540 Warning: Updating a style property during rerender (animation) when a conflicting property is set (animationPlayState) can lead to styling bugs. To avoid this, don't mix shorthand and non-shorthand properties for the same value; instead, replace the shorthand with separate values.
    at div
    at section
    at div
    at Marquee (http://localhost:3000/_next/static/chunks/pages/index.js?ts=1637510768241:6885:19)
    at div
    at Home (http://localhost:3000/_next/static/chunks/pages/index.js?ts=1637510768241:5988:66)
    at div
    at div
    at Layout (http://localhost:3000/_next/static/chunks/pages/_app.js?ts=1637510768241:526:23)
    at MyApp (http://localhost:3000/_next/static/chunks/pages/_app.js?ts=1637510768241:1503:24)
    at ErrorBoundary (http://localhost:3000/_next/static/chunks/main.js?ts=1637510768241:764:47)
    at ReactDevOverlay (http://localhost:3000/_next/static/chunks/main.js?ts=1637510768241:880:23)
    at Container (http://localhost:3000/_next/static/chunks/main.js?ts=1637510768241:8865:5)
    at AppContainer (http://localhost:3000/_next/static/chunks/main.js?ts=1637510768241:9361:24)
    at Root (http://localhost:3000/_next/static/chunks/main.js?ts=1637510768241:9500:25)
```

**Comment by Lucien950** [comment]
More context: This happens every time I run useEffect

**Comment by Lucien950** [comment]
I believe if you change line 60 and 61 at `index.tsx` and put them together, it will fix it.

**Comment by jagnani73** [comment]
@Lucien950 hi!
If you're willing to, I will assign this issue to you and you're more than welcome to debug it!

I apologise for the my late response!

**Comment by Lucien950** [comment]
I don't know how to fix

**Comment by jagnani73** [comment]
Its okay @Lucien950 
Its now fixed in #16
